controller/src/main/java/io/pravega/controller/server/rest/resources/StreamMetadataResourceImpl.java

```diff
@@ -301,11 +302,8 @@ public void getReaderGroup(final String scopeName, final String readerGroupName,
                                final SecurityContext securityContext, final AsyncResponse asyncResponse) {
         long traceId = LoggerHelpers.traceEnter(log, "getReaderGroup");
 
-        try {
-            authenticate(scopeName + "/" + readerGroupName, READ);
-        } catch (AuthenticationException e) {
+        if (!authenticate(scopedStreamName(scopeName, readerGroupName), READ, asyncResponse)) {
             log.warn("Get reader group for {} failed due to authentication failure.", scopeName + "/" + readerGroupName);
```

Comment:
change `{}` to `{}/{}` and pass `scopeName` and `readerGroupName` as separate arguments. You want to do as little computation as possible in the log arguments.

Reply:
Fixed.